src/utils/errors/ApiError.ts

```
@@ -0,0 +1,60 @@
+    details?: unknown
+  ) {
+    super(message);
+    this.name = "ApiError";
+    this.code = code;
+    this.status = status;
+    this.details = details;
+    Object.setPrototypeOf(this, ApiError.prototype);
+  }
+
+  static fromAxiosError(error: AxiosError): ApiError {
+    const message = (error.response?.data as any)?.message || error.message || "알 수 없는 오류가 발생했습니다.";
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)